src/test/rustdoc/html_source.rs

```diff
@@ -0,0 +1,4 @@
+#![crate_name = "foo"]
+
```

Comment:
Seems like it might be worth adding:

```suggestion

// @has src/foo/html_source.rs.html
```

Reply:
Indeed! I added it too.